paddle/fluid/operators/nccl_op_test.cu.cc

```diff
@@ -121,27 +134,11 @@ class NCCLTester : public ::testing::Test {
   std::vector<p::DeviceContext *> dev_ctxs;
   f::Scope g_scope;
   std::mutex mu;
+  std::vector<int> gpu_list;
```

Comment:
Data members of classes should be with a trailing underscore. 
refer : [Variable Names](https://google.github.io/styleguide/cppguide.html#Variable_Names)